docs/service_architecture.md

```diff
@@ -12,12 +12,63 @@ weight=3
 
 # Understand the Notary service architecture
 
-On this page, you get an overview of the Notary service architecture. This
-document assumes a prior understanding of [The Update
-Framework](https://theupdateframework.github.io/).
+On this page, you get an overview of the Notary service architecture.
+
+## Brief overview of TUF keys and roles
+
+This document assumes familiarity with [The Update Framework](https://theupdateframework.github.io/),
+but here is a brief recap of the TUF roles and corresponding key hierarchy:
+
+<center><img src="images/key-hierarchy.svg" alt="TUF Key Hierarchy" width="400px"/></center>
+
+- The root key is the root of all trust.  It signs the
+[root metadata file](https://github.com/theupdateframework/tuf/blob/develop/docs/tuf-spec.txt#L489),
```

Comment:
Probably better to change these links to a fixed commit, to prevent them being outdated in future, I.e. https://github.com/theupdateframework/tuf/blob/1bed3e09a478c2c918ffbff10b9118f6e52ee129/docs/tuf-spec.txt#L489 in this case